app/assets/javascripts/discourse/app/lib/utilities.js

```diff
@@ -320,16 +317,14 @@ export function isiPad() {
 }
 
 export function safariHacksDisabled() {
-  if (iOSWithVisualViewport()) {
-    return false;
-  }
+  deprecated(
+    "`safariHacksDisabled()` is deprecated, it now always returns `false`",
+    {
+      since: "2.8.0.beta8",
```

Comment:
Can we mark this for removal in next release?

Reply:
tbh I don't see the point of `dropFrom` in these notices, since we never actually remove them when we said we will 😂

Reply:
Well, `dropFrom` indicates to consumers that past a defined version the function _can_ be removed. And the next person that goes through the `dropFrom`s can more easily do the actual dropping. (Without a version, we're less sure.)

Reply:
I'd say the assumed default `dropFrom` is the first beta after a stable release. (the logic being: we can't remove it from the next stable release, because it would be pulling a rug from under users that stick to stable releases only, but the next beta release after that is fair play)

Maybe `deprecated()` could calculate that `dropFrom` version if the option wasn't passed in?

Anyway, having that version number there doesn't make the removal easier, since we first do have to check and (almost always) fix the remaining uses of a deprecated feature in all-the* plugins/theme components.